12-add-tupletools.md

```diff
@@ -53,18 +53,18 @@ The way the `DecayTreeTuple.Decay` is written in in our [minimal DaVinci job](08
 dtt.Decay = '[D*(2010)+ -> (D0 -> K- pi+) pi+]CC'
 ```
 
-means that the configured `TupleTools` will only run on the head of the decay chain, that is, the D*.
+means that the configured `TupleTools` will only run on the head of the decay chain, that is, the D\*.
 In order to select the particles for which we want the information stored, we need to mark them with a `^` symbol in the decay descriptor.
-For example, if we want to fill the information of the D0 and its children, we would modify the `dtt` to look like this:
+For example, if we want to fill the information of the D0 and all its children, we would modify the `dtt` to look like this:
```

Comment:
When the soft pion wasn't marked this statement was true, now you'll need to change it to “…fill the information of the D*+ and all of it's children, …”.